### ISSUE TITLE
Kabanero operator should set up Codewind tekton roles

### ISSUE THREAD

**Issue by johnmcollier** [kabanero-io/kabanero-operator#536]
(Originally https://github.com/eclipse/codewind/issues/2333)

In order to properly launch the tekton dashboard from the "Open Tekton Dashboard" button in Codewind/Theia, Codewind needs to be set up with a cluster role and rolebinding allow it to retrieve the tekton dashboard service in the tekton-pipelines namespace.

You can see the files here:
- https://github.com/eclipse/codewind-che-plugin/blob/master/setup/install_che/codewind-tektonrole.yaml
- https://github.com/eclipse/codewind-che-plugin/blob/master/setup/install_che/codewind-tektonbinding.yaml

This issue isn't a stopship and so my fix (https://github.com/kabanero-io/kabanero-operator/pull/535) should only be merged after release.

**Comment by mtamboli** [comment]
@kaczyns where would this defect be fixed: install script or kabanero operator?

**Comment by jgawor** [comment]
@johnmcollier is there a work-around that doesn't require operator updates?

**Comment by johnmcollier** [comment]
@jgawor The following command could be used as a workaround until the operator is updated:
```
kubectl apply -f https://gist.githubusercontent.com/johnmcollier/41031692b8bfe067ef28223069e57871/raw/00cf207e1f9f3b6046de61b278b7b5f96f1a92b0/kabanero-codewind-tekton.yaml
```

**Note:** We would want to move the yaml off of a github gist and somewhere more permanent.

**Comment by marikaj123** [comment]
According to Jarek, this issue does not require to be fixed in 0.6.0.

**Comment by marikaj123** [comment]
@johnmcollier - Does Codewind/CodeReady workspace have the fix? If yes, what would be the version that would contain the fix?

**Comment by johnmcollier** [comment]
@marikaj123 Not yet, it's a fix I need to make in the Kabanero operator. I can deliver it for Kabanero 0.9.0

**Comment by marikaj123** [comment]
@johnmcollier - Thank you for getting it fixed for Kabanero 0.9.0.  The fix will need to be complete during the month of April to make it into Kabanero 0.9.0.

**Comment by marikaj123** [comment]
@johnmcollier -  Are you producing the che-devfile-registry image that in turn pulls in the correct version of codewind to allow the operator to include the codewind 0.11.0?
